ClientLibrary/Samples/Work/BoardsSample.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.VisualStudio.Services.WebApi;
+using Microsoft.TeamFoundation.Core.WebApi.Types;
+using Microsoft.TeamFoundation.Work.WebApi;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.TeamFoundation.Core.WebApi;
+
+namespace Microsoft.Azure.DevOps.ClientSamples.Work
+{
+    [ClientSample(WorkWebConstants.RestArea, "boards")]
+    public class BoardsSample : ClientSample
+    {
+        [ClientSampleMethod]
+        public Board GetTeamStoriesBoard()
+        {
+            VssConnection connection = Context.Connection;
+            WorkHttpClient workClient = connection.GetClient<WorkHttpClient>();
+
+            TeamProjectReference project = ClientSampleHelpers.FindAnyProject(this.Context);
+            WebApiTeamRef team = ClientSampleHelpers.FindAnyTeam(this.Context, project.Id);
+
+            TeamContext context = new TeamContext(project.Id, team.Id);
+            context.Team = team.Name;
+            context.Project = project.Name;          
+
+            Board board = workClient.GetBoardAsync(context, "Stories").Result;
```

Comment:
Is `Stories` a special keyword? If so, is there a constant variable (in the public client) that can be referenced here instead?

Reply:
"Stories" is just a work item type and can change depending on the process. You can make it a constant but that won't really help :(